src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -6184,6 +6184,7 @@ public bool Equals(TypedConstant x, TypedConstant y)
                    x.Kind switch
                    {
                        TypedConstantKind.Array => x.Values.SequenceEqual(y.Values, TypedConstantComparer.Instance),
+                       TypedConstantKind.Type => TypesEquivalent(x.Value as ITypeSymbol, y.Value as ITypeSymbol, exact: true),
```

Comment:
>exact [](http://example.com/codeflow?start=113&length=5)

This should probably be `exact: false`. E.g. if the types are `typeof(object)` and `typeof(dynamic)` the attribute data will be the same.